src/main/java/de/tum/in/www1/artemis/service/dto/UserDTO.java

```diff
@@ -39,9 +39,7 @@
     @Size(max = 50)
     private String lastName;
 
-    @Email
-    @Size(min = 5, max = 100)
-    private String email;
+    private String visibleEmail;
```

Comment:
Why exactly did you change it here as well? Imo we could leave the variable name here.
But at the very least the Constraints should stay.

This would make sense/be necessary, as the UserDTO is also used when editing/creating users on the client. So having the constraints is strictly necessary and calling it `visibleEmail` is slightly confusing.

Reply:
Hmm, didn't give it too much thought. I made it consistent with the registration number and it's called visibleRegistrationNumber in the client too.

But I agree, it might make sense to invert this. *sigh* :D